spec.md

```diff
@@ -644,6 +644,7 @@ message PluginCapability {
   oneof type {
     // Service that the plugin supports.
     Service service = 1;
+    // deprecated
```

Comment:
"deprecated" should also be added in front of the "message VolumeExpansion" on line 598 as well?

Reply:
fixed